src/chainlit/session.py

```diff
@@ -45,7 +46,7 @@ def __init__(
         self.db_client = db_client
         self.should_stop = False
         self.restored = False
-        self.id = token_urlsafe()
```

Comment:
I think the main goal of token url safe is to share a "secured" secret while the main goal of uuid is to generate unique identifiers. Since the session id will be stored in a httpOnly cookie, we are less interested in the "secure" property than the "unique" property.

Might be wrong on that though.

Reply:
I agree but still, that's a very cheap way to improve security IMO, as I don't see any reason to prefer an UUID

Reply:
(that being said I'm fine with both honestly)